cmd/sovereignnode/chainSimulator/tests/bridge/bridgeChaos_test.go

```diff
@@ -0,0 +1,285 @@
+			RoundsPerEpoch:         core.OptionalUint64{},
+			ApiInterface:           api.NewNoApiInterface(),
+			MinNodesPerShard:       2,
+			AlterConfigsFunction: func(cfg *config.Configs) {
+				cfg.SystemSCConfig.ESDTSystemSCConfig.BaseIssuingCost = issuePaymentCost
+				cfg.GeneralConfig.SovereignConfig.OutgoingSubscribedEvents.SubscribedEvents = []config.SubscribedEvent{
+					{
+						Identifier: "deposit",
+						Addresses:  []string{outGoingSubscribedAddress},
+					},
+				}
+				cfg.GeneralConfig.SovereignConfig.OutgoingSubscribedEvents.TimeToWaitForUnconfirmedOutGoingOperationInSeconds = 1
+				cfg.GeneralConfig.VirtualMachine.Execution.TransferAndExecuteByUserAddresses = []string{outGoingSubscribedAddress}
+			},
+		},
+	})
+	require.Nil(t, err)
+	require.NotNil(t, cs)
+
+	defer cs.Close()
+
+	err = cs.GenerateBlocks(1)
+	require.Nil(t, err)
+
+	nodeHandler := cs.GetNodeHandler(core.SovereignChainShardId)
+
+	initialAddress := "erd1l6xt0rqlyzw56a3k8xwwshq2dcjwy3q9cppucvqsmdyw8r98dz3sae0kxl"
+	initialAddrBytes, err := nodeHandler.GetCoreComponents().AddressPubKeyConverter().Decode(initialAddress)
+	require.Nil(t, err)
+
+	err = cs.SetStateMultiple([]*dtos.AddressState{
+		{
+			Address: initialAddress,
+			Balance: "10000000000000000000000",
+		},
+	})
+	require.Nil(t, err)
+
+	err = cs.GenerateBlocks(1)
+	require.Nil(t, err)
+
+	wallet := dtos.WalletAddress{Bech32: initialAddress, Bytes: initialAddrBytes}
+
+	expectedESDTSafeAddressBytes, err := nodeHandler.GetCoreComponents().AddressPubKeyConverter().Decode(outGoingSubscribedAddress)
+	require.Nil(t, err)
+
+	bridgeData := deploySovereignBridgeSetup(t, cs, wallet, esdtSafeWasmPath, feeMarketWasmPath)
+	require.Equal(t, expectedESDTSafeAddressBytes, bridgeData.ESDTSafeAddress)
+
+	nonce := GetNonce(t, nodeHandler, wallet.Bech32)
+
+	// Issue new fungible token
+	issueCost, _ := big.NewInt(0).SetString(issuePaymentCost, 10)
+	tokenIdentifier := chainSim.IssueFungible(t, cs, wallet.Bytes, &nonce, issueCost, "SovToken", "SVN", 18, big.NewInt(100000))
+
+	creator, _ := hex.DecodeString("0000000000000000000000000000000000000000000000000000000000000000")
+	esdtToken := sovereign.EsdtToken{
+		Identifier: []byte(tokenIdentifier),
+		Nonce:      0,
+		Data: sovereign.EsdtTokenData{
+			TokenType:  core.Fungible,
+			Amount:     big.NewInt(1),
+			Frozen:     false,
+			Hash:       []byte(""),
+			Name:       []byte(""),
+			Attributes: []byte(""),
+			Creator:    creator,
+			Royalties:  big.NewInt(0),
+			Uris:       [][]byte{},
+		},
+	}
+
+	currentOpNonce := uint64(0)
+	numOfTransfers := 10
+	sovereignOutGoingOps := make([]sovereign.Operation, 0)
+
+	// Deposit tokens
+	operations := depositEsdtTokenMultipleTimes(t, cs, esdtToken, wallet, &nonce, bridgeData.ESDTSafeAddress, &currentOpNonce, numOfTransfers)
+	sovereignOutGoingOps = append(sovereignOutGoingOps, operations...)
+
+	serializer, _ := abi.NewSerializer(abi.ArgsNewSerializer{
+		PartsSeparator: "@",
+	})
```

Comment:
You can delete it, you already have a shadowed variable
```suggestion
	serializer, _ := abi.NewSerializer(abi.ArgsNewSerializer{
		PartsSeparator: "@",
	})
```

Reply:
right